index.js

```diff
@@ -1,37 +1,38 @@
-var HtmlbarsMinifier = require('./htmlbars-minifier');
+var HtmlbarsMinifier = require('./lib/htmlbars-minifier');
 
 module.exports = {
-    name: 'ember-cli-htmlbars-minifier',
+	name: 'ember-cli-htmlbars-minifier',
 
-    initializeOptions: function(appOptions) {
-        var defaultOptions = {
-            stripIndentation: true,
-            removeTrailingSpaces: true,
-            coalesceSpaces: true,
-            removeSpacesAroundTags: true,
-            stripNewlines: true,
-        };
+	initializeOptions: function(appOptions) {
+		var defaultOptions = {
+			stripIndentation: true,
+			removeTrailingSpaces: true,
+			coalesceSpaces: true,
+			removeSpacesAroundTags: true,
+			stripNewlines: true,
+		};
 
-        this.options = appOptions["ember-cli-htmlbars-minifier"] || {};
-        for (var option in defaultOptions) {
-            if (!this.options.hasOwnProperty(option)) {
-                this.options[option] = defaultOptions[option];
-            }
-        }
-    },
-    included: function(app) {
-        this._super.included.apply(this, arguments);
+		this.options = appOptions['ember-cli-htmlbars-minifier'] || {};
+		for (var option in defaultOptions) {
+			if (!this.options.hasOwnProperty(option)) {
+				this.options[option] = defaultOptions[option];
+			}
+		}
+	},
+	included: function(app) {
+		this._super.included.apply(this, arguments);
```

Comment:
you are initializing config so I think It's better on config hook.  [Docs](https://ember-cli.com/api/classes/Addon.html) said included hook would typically be used to perform additional imports"